app/proguard-rules.txt

```diff
@@ -15,7 +15,7 @@
 # unnecessarily. When castParam() is needed, it's not stripped. e.g.:
 #
 #  @OnItemSelected(value = R.id.history_entry_list)
-#  void foo(ListView bar) {
+#  void showImageWithItem(ListView bar) {
```

Comment:
un-related, must have happened while refactoring some other field

Reply:
Yes, it was a result of refactoring. I'll revert in the next commit